travis-ymls/mothur.travis.yml

```diff
@@ -0,0 +1,36 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : mothur
+# Source Repo         : https://github.com/ujjwalsh/mothur
+# Travis Job Link     : https://travis-ci.com/github/ujjwalsh/mothur/builds/213262386
+# Created travis.yml  : N
+# Maintainer          : Ujjwal Sharma
+#
+# Script License      : g++
```

Comment:
This should be always `Apache License, Version 2 or later`.

Reply:
done